app/Core/Policies/DirectMessagePolicy.php

```diff
@@ -19,6 +19,6 @@ class DirectMessagePolicy
      */
     public function delete(User $user, DirectMessage $directMessage)
     {
-        return $user->id == $directMessage->sender_id;
+        return $user->id === $directMessage->sender_id;
```

Comment:
@Hasnayeen sadly it seems the type is not same (int vs string)

Reply:
@RageZBla oh ok, that's weird. revert back to original then.